Minor TA wording improvement (#18996)
cockroachdb/docs: src/current/advisories/a131639.md

## Description

In v22.2 through v24.2, in rare circumstances where there is a sustained period of disk slowness in the presence of lease transfers, it is possible for some writes in a transaction that straddle multiple ranges to be lost. In cases where these lost writes are not also present on one or more secondary indexes that could allow for reconstructing or repairing the row, these writes could be irrecoverably lost.

> Single-range writes, implicit transactions, and workloads using the One-Phase-Commit (1PC) fast path are not affected.

Reviewer: Saying "implicit transactions" here is NOT correct. Let's remove it and just say:

"Transactions that write to a single range, or transactions that take advantage of the One-Phase-Commit (1PC) optimization, are not affected."

For context, saying "implicit transactions" is not correct, because you can have something like an `INSERT ... VALUES (a), (b);` that straddles multiple ranges, or simply an `INSERT ... VALUES (a)` that needs to update secondary indexes.

> Single-range writes, implicit transactions, and workloads using the One-Phase-Commit (1PC) fast path are not affected.

Author: SGTM. Thanks. Just adding as a suggestion here for quick implementation.

```suggestion
Transactions that write to a single range, or transactions that take advantage of the One-Phase-Commit (1PC) optimization, are not affected.
```
